src/ConnectedMode/UI/Resources/Styles.xaml

```diff
@@ -7,6 +7,16 @@
         <Setter Property="MinWidth" Value="75" />
     </Style>
 
+    <Style TargetType="TextBox">
```

Comment:
Should we add theming here? Later maybe?

Reply:
I would say as part of: https://sonarsource.atlassian.net/browse/SLVS-1340